src/vue/forms/CreateIssuanceForm.vue

```diff
@@ -0,0 +1,270 @@
+
+const EVENTS = {
+  closeDrawer: 'close-drawer'
+}
+
+export default {
+  name: 'create-issuance-form',
+  mixins: [FormMixin],
+  data: () => ({
+    form: {
+      assetCode: '',
+      amount: null,
+      receiver: '',
+      reference: ''
+    },
+    selectedTokenAvailableToIssuance: null,
+    ACCOUNT_TYPES,
+    accountOwnedTokens: [],
+    accountOwnedTokensCodes: []
+  }),
+  validations: {
+    form: {
+      assetCode: { required },
+      amount: {
+        required,
+        amount
+        // TODO: fix it
+        // maxValueWrapper: maxValueWrapper(() => {
+        //   console.log('this', this)
+        //   return +this.selectedTokenAvailableToIssuance
+        // })
+      },
+      receiver: { required, email },
+      reference: { required }
+    }
+  },
+  computed: {
+    ...mapGetters([
+      vuexTypes.accountTypeI,
+      vuexTypes.walletEmail,
+      vuexTypes.accountId
+    ])
+  },
+  watch: {
+    'form.assetCode' (value) {
+      if (value) this.getAvailableToIssuance(value)
+    },
+    'accountOwnedTokensCodes' (value) {
+      this.setTokenCode()
+    }
+  },
+  created () {
+    this.getAccountOwnedTokens()
+    this.setTokenCode()
+    if (this.form.assetCode) {
+      this.getAvailableToIssuance(this.form.assetCode)
+    }
+  },
+  methods: {
+    async getAccountOwnedTokens () {
+      const tokens = await this.loadAllTokens()
+      const accountOwnedTokens = tokens.filter(i => i.owner === this.accountId)
+
+      this.accountOwnedTokens = accountOwnedTokens
+      this.accountOwnedTokensCodes = accountOwnedTokens.map(i => i.code)
+    },
+    async submit () {
+      if (!await this.isFormValid()) return
+      this.disableForm()
+      try {
+        const receiver = await this.loadBalanceIdByEmailAndCode(
+          this.form.receiver,
+          this.form.assetCode
+        )
+        // error message for user with information about why receiver isn't
+        // present inside loadBalanceIdByEmailAndCode method
+        if (!receiver) return false
+        const operation =
+          await base.CreateIssuanceRequestBuilder.createIssuanceRequest({
+            asset: this.form.assetCode,
+            amount: this.form.amount,
+            receiver: receiver,
+            reference: this.form.reference,
+            externalDetails: {}
+          })
+        await Sdk.horizon.transactions.submitOperations(operation)
+        Bus.success(globalize('create-issuance-form.issuance-succeeded'))
+        this.$emit(EVENTS.closeDrawer)
+      } catch (error) {
+        if (error.constructor === errors.NotFoundError) {
+          Bus.error(
+            globalize('create-issuance-form.no-balance', {
+              asset: this.form.assetCode
+            })
+          )
+          this.enableForm()
+          return
+        }
+        ErrorHandler.process(error)
+      }
+      this.enableForm()
+    },
+
+    async loadBalanceIdByEmailAndCode (email, assetCode) {
+      // TODO: add sign
```

Comment:
I remember you promised to check if there autosign in js-sdk before creating the PR?